sw/device/tests/rstmgr_cpu_info_test.c

```diff
@@ -24,176 +23,253 @@ OTTF_DEFINE_TEST_CONFIG();
  * RSTMGR CPU INFO TEST
  *  This test creates a double fault by accessing a register with
  *  a non-existing address.
- *  After the double fault, the dut gets reset by the watch dog bite,
+ *  After a single fault and then a double fault,
+ *  the dut gets reset by the watch dog bite,
  *  and the test collects / checks the cpu_info from the rstmgr.
  */
 
-// non existing address
-#define kIllegalAddr1 0x4041FFF0u
-#define kIllegalAddr2 0x40003618u
-#define kSkipComp 0x12345678u
+// Unmapped Addresses
+static const uint8_t *kIllegalAddr0 = (uint8_t *)0xA0A0DEAFu;
+static const uint8_t *kIllegalAddr1 = (uint8_t *)0xA041FFF0u;
+static const uint8_t *kIllegalAddr2 = (uint8_t *)0xA0003618u;
```

Comment:
The `u` prefix is no longer needed. These could also be enums or preprocessor definitions, but maybe it's not worth changing that now. As discussed in person, they could also be `volatile` to avoid the assignment to `addr_val`, but since you have other uses for `addr_val` it's probably not worth changing that.